src/main/sensors/battery.c

```diff
@@ -434,6 +434,14 @@ uint8_t getBatteryCellCount(void)
     return batteryCellCount;
 }
 
+/**
+ * Gets average battery cell voltage in 0.01V units.
+ */
+uint16_t getBatteryAverageCellVoltage(void)
```

Comment:
I did consider that, but this is required to keep the 0.01V precision currently used for the average cell voltage in OSD (at least without using a `float` instead).

I could just keep this function in `osd.c`, that would keep the existing precision while maintaining consistency in `battery.c`?